extensions/amp-analytics/0.1/vendors/gtag.js

```diff
@@ -116,6 +116,27 @@ export const GTAG_CONFIG = /** @type {!JsonObject} */ ({
       'https://${flSrc}.fls.doubleclick.net/activityi;src=${flSrc};type=${flType};cat=${flCat}',
     'dnsBase': 'https://ad.doubleclick.net/ddm/clk/',
   },
+  'triggers': {
+    'storyProgress': {
+      'on': 'story-page-visible',
+      'enabled': false,
+      'request': 'uaEvent',
```

Comment:
Unfortunately, this isn't going to work. 
uaEvent is not fully defined here (at least tracking ID is missing). So we'll end up with malformed UA request sent on *any* gtag installation (even it's for Ads only)

Proper ways to do this are:
* Suggest customers to configure tracking on page:
```
"triggers": {
    "storyProgress": {
      "on": :story-page-visible",
      "vars": {
        "event_name": "custom",
        "event_action": "story_progress",
        "event_category": "${title}",
        "event_label": "${storyPageId}",
        "send_to": [
          "UA-123-1", // ID here
        ]        
      }
    }
  }
```
* Create native gtag support (say, have a special condition in config command that tells rewriter to generate this request). - Subject to prioritization on gtag side.